ceremonies/src/lib.rs

```diff
@@ -373,7 +373,9 @@ impl<T: Config> Module<T> {
         for cid in cids.iter() {
             let pcount = <ParticipantCount>::get((cid, cindex));
             let ecount = <EndorseesCount>::get((cid, cindex));
-            let n_locations = <encointer_communities::Module<T>>::locations(cid).len();
+            // FIXME
```

Comment:
yep, we will have to talk about this in more detail, because location assignment and issuance depend on the `communities` pallet and therefore our design changes here will have implications.

Reply:
sure. the PR may not break anything, as we only ,erge what passes CI tests